manage.py

```diff
@@ -315,5 +318,28 @@ def db_seed():
     # counts objects seeded
     print(f'obj count: {len(db.session.query(Applicant).all())}')
 
+@manager.command
+def test():
```

Comment:
@rrabinovitch im curious what these two new commands are in here for because from the directions you added to the readme concerning running tests with and without coverage, neither of those include these new `manage.py test` and `manage.py cov` commands.